src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
> To reproduce you can change the databasename in the config.yml to a non existing databasename and you should get the error.

That is exactly what I was doing, with `debug: nuclear` … joking about the "nuclear", that is -1 (`E_ALL`). :laughing: